deploy.sh

```diff
@@ -40,6 +43,28 @@ do
     esac
 done
 
+# external SCMs config
+if $xflag
+then
+    if $dflag; then echo "Demo mode not compatible with -x option"; exit 1; fi #validate no demo
+    declare -a SCMS=(ado bb github gitlab) # declare all the external SCMs supported
+    if [[ " ${SCMS[@]} " =~ " ${SCM} " ]]; then
+        case ${SCM} in
+            "ado" )
+                SCM_DIR=${DIRNAME}/thirdparty-scms/ado
```

Comment:
Would it make sense to use:
`SCM_DIR=${DIRNAME}/thirdparty-scms/${SCM}`
?

Reply:
No problem with this. I'll do it